packages/react-core/src/components/DualListSelector/examples/DualListSelector.md

```diff
@@ -0,0 +1,194 @@
+---
+id: Dual list selector
+section: components
+cssPrefix: 'pf-c-dual-list-selector'
+propComponents: ['DualListSelector']
+beta: true
+---
+
+import PficonSortCommonDescIcon from '@patternfly/react-icons/dist/js/icons/pficon-sort-common-desc-icon';
+import PficonSortCommonAscIcon from '@patternfly/react-icons/dist/js/icons/pficon-sort-common-asc-icon';
+
+## Examples
+
+### Basic
+
+```js
+import React from 'react';
+import { 
+  DualListSelector
+} from '@patternfly/react-core';
+
+class BasicDualListSelector extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      availableOptions: ['Option 1', 'Option 2', 'Option 3', 'Option 4'],
+      chosenOptions: []
+    };
+    
+    this.onListChange = (newAvailableOptions, newChosenOptions) => {
+      this.setState({
+        availableOptions: newAvailableOptions.sort(),
+        chosenOptions: newChosenOptions.sort(),
+      })
+    };
+  }
+  
+  render() {
+    return (
+      <DualListSelector
+        availableOptions={this.state.availableOptions}
+        chosenOptions={this.state.chosenOptions}
+        onListChange={this.onListChange}
+        id="basicSelector"
+      />
+    );
+  }
+}
+
+```
+
+### Basic with actions
+
+```js
+import React from 'react';
+import { 
+  Button,
+  ButtonVariant,
+  Dropdown,
+  DropdownItem,
+  DualListSelector,
+  KebabToggle
+} from '@patternfly/react-core';
+import PficonSortCommonDescIcon from '@patternfly/react-icons/dist/js/icons/pficon-sort-common-desc-icon';
+import PficonSortCommonAscIcon from '@patternfly/react-icons/dist/js/icons/pficon-sort-common-asc-icon';
+
+class ManagedDualListSelector extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      availableOptions: [
+        <span>Option 1</span>, 
```

Comment:
Curious why these are in a span?

Reply:
The most common case I expect will be that the options will all be strings, but the type of the prop is an array of ReactNodes, so this example is showing how you can wire up all the pieces to work if you are passing something more complicated that a string.